src/database/schema.js

```diff
@@ -124,36 +140,41 @@ User.schema = {
 export const schema = {
   schema: [
     Address,
+    IndicatorAttribute,
+    IndicatorValue,
     Item,
     ItemBatch,
-    ItemDepartment,
     ItemCategory,
+    ItemDepartment,
     ItemStoreJoin,
-    Transaction,
-    TransactionItem,
-    TransactionBatch,
-    TransactionCategory,
     MasterList,
     MasterListItem,
     MasterListNameJoin,
+    Message,
     Name,
     NameStoreJoin,
     NumberSequence,
     NumberToReuse,
     Options,
+    Report,
+    Period,
+    PeriodSchedule,
+    ProgramIndicator,
     Requisition,
     RequisitionItem,
     Setting,
-    SyncOut,
     Stocktake,
-    StocktakeItem,
-    Period,
-    PeriodSchedule,
     StocktakeBatch,
-    User,
+    StocktakeItem,
+    SyncOut,
+    Transaction,
+    TransactionBatch,
+    TransactionCategory,
+    TransactionItem,
     Unit,
+    User,
   ],
-  schemaVersion: 10,
+  schemaVersion: 13,
```

Comment:
think we should change this to 11

Reply:
a7bbf5d63170944846942d5e0da84b2f89536d52.